incubator/hnc/internal/kubectl/tree.go

```diff
@@ -81,6 +84,21 @@ func printSubtree(prefix string, hier *api.HierarchyConfiguration, inCycle bool)
 		if cycle && inCycle {
 			continue
 		}
+
+		ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+		defer cancel()
+
+		ns, err := k8sClient.CoreV1().Namespaces().Get(ctx, txt, metav1.GetOptions{})
+		if err != nil {
+			fmt.Printf("Could not get namespaces: %s\n", err)
+			os.Exit(1)
+		}
+
+		_, hasSubnamespaceAnnotation := ns.ObjectMeta.Annotations[api.SubnamespaceOf]
+		if hasSubnamespaceAnnotation {
```

Comment:
Go prefers short variable names if the name's only used once or twice. So you could use the following here:

```
// Identify subnamespaces
if _, ok := ns.ObjectMeta.Annotations[api.SubnamespaceOf]; ok {
  txt = "[s] " + txt
}
``

Reply:
done